monkey/tests/unit_tests/infection_monkey/credential_store/test_aggregating_credentials_store.py

```diff
@@ -0,0 +1,95 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from infection_monkey.credential_collectors import Password, SSHKeypair, Username
+from infection_monkey.credential_store import AggregatingCredentialsStore
+from infection_monkey.i_puppet import Credentials
+
+CONTROL_CHANNEL_CREDENTIALS = {
+    "exploit_user_list": ["Administrator", "root", "user1"],
+    "exploit_password_list": ["123456", "123456789", "password", "root"],
+    "exploit_lm_hash_list": ["aasdf23asd1fdaasadasdfas"],
+    "exploit_ntlm_hash_list": ["asdfadvxvsdftw3e3421234123412", "qw4trklxklvznksbhasd1231"],
+    "exploit_ssh_keys": [
+        {"public_key": "some_public_key", "private_key": "some_private_key"},
+        {
+            "public_key": "ssh-ed25519 AAAAC3NzEIFaJ7xH+Yoxd\n",
+            "private_key": "-----BEGIN OPENSSH PRIVATE KEY-----\nb3BdHIAAAAGYXjl0j66VAKruPEKjS3A=\n"
+            "-----END OPENSSH PRIVATE KEY-----\n",
+        },
+    ],
+}
+
+CREDENTIALS_COLLECTION = [
+    Credentials(
+        [Username("user1"), Username("user3")],
+        [
+            Password("abcdefg"),
+            Password("root"),
+            SSHKeypair(public_key="some_public_key_1", private_key="some_private_key_1"),
+        ],
+    )
+]
+
+SAME_SSH_KEYPAIR = [
```

Comment:
What does this name mean? The same as what?

Reply:
Really bad naming, right? I want to check in the tests that if I have same ssh_keypair, it will check and not add it.